storage_service/locations/api/resources.py

```diff
@@ -769,3 +771,57 @@ def _attempt_package_request_event(self, package, request_info, event_type, even
             status_code = 200
 
         return (status_code, response)
+
+    @_custom_endpoint(expected_methods=['get'])
+    def package_contents(self, request, bundle, **kwargs):
+        response = {
+            "success": True,
+            "package": bundle.obj.uuid,
+            "files": []
+        }
+
+        for f in bundle.obj.file_set.all():
+            response["files"].append({attr: getattr(f, attr) for attr in ('source_id', 'name', 'source_package', 'checksum', 'accessionid', 'origin')})
+
+        return http.HttpResponse(status=200, content=json.dumps(response),
+            content_type='application/json')
+
+    def file_data(self, request, **kwargs):
```

Comment:
Docstrings for these? I know we don't have them on other functions, but they sure are nice.